scg-docker/Dockerfile.telicent

```diff
@@ -0,0 +1,49 @@
+## Dockerfile for Smart Cache Graph
+## Uses Telicent Base Image
```

Comment:
It does? Am I being completely blind or does this `Dockerfile` not just self-reference the normal Docker image??